scripts/api-html-artifacts.json

```diff
@@ -1,5 +1,6 @@
 {
   "qiskit": {
+    "1.0-dev": "https://ibm.box.com/shared/static/eh9tzx3senx5xkq1vvtsr5h697r6j7gk.zip",
```

Comment:
What about simplifying this to be `dev`? While in Box I think the file should be called `1.0-dev`, it doesn't need to be so here. That avoids the confusion when we add `1.1-dev` - what do we do with `1.0-dev`? It wouldn't make sense to keep it here. It's only possible to have a single `dev` version in this repository.

The motivation for Box to name `1.0-dev` is so that we can have a historical record of our past dev docs.

Reply:
That's much better, thank you!